src/test/java/com/epam/tc/hw1/TestDivide.java

```diff
@@ -0,0 +1,21 @@
+package com.epam.tc.hw1;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.annotations.Test;
+
+public class TestDivide {
+
+    @Test(
+            dataProviderClass = DataProviderForCalculator.class,
+            dataProvider = "Data for divide"
+    )
+    void testDivideLong(long a, long b, long expected) {
+
+        Calculator calculator = new Calculator();
```

Comment:
Please move initialization if SUT (system under test) to dedicated method
This method is common for all your test classes, so extract this method to some parent class